pkg/serverless/executioncontext/executioncontext_test.go

```diff
@@ -60,6 +60,17 @@ func TestSetFromInvocationWarmStart(t *testing.T) {
 	assert.Equal(false, ec.coldstart)
 }
 
+func TestSetArnFromExtensionResponse(t *testing.T) {
+	assert := assert.New(t)
+
+	testArn := "arn:aws:lambda:us-east-1:123456789012:function:MY-SUPER-function"
```

Comment:
do we need to take care about the version (last part of the arn?)

Reply:
typically not in this case, since we're building the ARN manually, versus using the ARN typically received from the `INVOKE` event that will have the version